sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/implementation/Utility.java

```diff
@@ -1348,4 +1349,15 @@ public static SentenceAssessment findSentimentAssessment(String assessmentPointe
         }
         return assessments.get(assessmentIndex);
     }
+
+    public static void throwIfTargetServiceVersionFound(TextAnalyticsServiceVersion sourceVersion,
+                                                        List<TextAnalyticsServiceVersion> targetVersions,
```

Comment:
It is a static variable, so it will be one object(eg, V3.0) for TextAnalyticsServiceVersion class. It won't create a new object TextAnalyticsServiceVersion.V3_0 for each call.